src/OSPSuite.Core/Domain/Services/QuantityValuesUpdater.cs

```diff
@@ -111,24 +111,44 @@ private IParameter getOrAddModelParameter(ValueUpdaterParams valueUpdater, Param
          parameter = _parameterValueToParameterMapper.MapFrom(parameterValue);
 
          simulationBuilder.AddBuilderReference(parameter, parameterValue);
+
+         //now we remove the parameter by name from the parent container just in case it already exists but with a different type (distributed vs not distributed)
+         var potentialParameter = parentContainer.Parameter(parameterValue.Name);
```

Comment:
before we were only landing in this code if a parameter did not exist. Now we also land there if it exists but with a different type